liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/utils/ExecuteMojoUtil.java

```diff
@@ -0,0 +1,311 @@
+package io.openliberty.tools.maven.utils;
+
+import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.name;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.maven.model.Plugin;
+import org.apache.maven.model.PluginExecution;
+import org.apache.maven.plugin.logging.Log;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+import org.twdata.maven.mojoexecutor.MojoExecutor.Element;
+
+public class ExecuteMojoUtil {
+
+    // https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html
+    private static final ArrayList<String> compileParams = new ArrayList<>(Arrays.asList(
```

Comment:
Use naming convention for `static final` constants i.e. uppercase and underscores.